pkg/agent/openflow/packetin.go

```diff
@@ -50,16 +51,37 @@ func (c *client) StartPacketInHandler(stopCh <-chan struct{}) {
 	err := c.SubscribePacketIn(uint8(ofprAction), ch)
 	if err != nil {
 		klog.Errorf("Subscribe PacketIn failed %+v", err)
+		return
 	}
+	packetInQueue := workqueue.NewNamed("packetIn")
+	go c.parsePacketIn(packetInQueue, stopCh)
 
-	wait.PollUntil(time.Second, func() (done bool, err error) {
+	wait.PollUntil(time.Millisecond, func() (done bool, err error) {
```

Comment:
I am a bit confused, why didn't we use a select statement here?

```
for {
    select {
         case pktIn := <- ch:
             packetInQueue.Add(pktIn)
         case <- stpCh:
              break
    }
}

Reply:
Fixed